app/components/stepper-signup.hbs

```diff
@@ -0,0 +1,5 @@
+{{#if (eq this.currentStep 0)}}
+  <h1>Hello world</h1>

```

Comment:
What is this for :)

Reply:
Done with changes can you please check my latest commit